### ISSUE TITLE
[Documentation] Game Server, which are Shutdown are not problems of emulating

### ISSUE THREAD

**Issue by ghost** [PCSX2/pcsx2#526]
To the Bug Squad Team or Bositman,

There are 2 Games in PCSX2 Wikia, which are under "broken Games" or "Ingame Games", which would normally work, but which don't, because the servers of these games are shutted down.

This is not the problem of PCSX2, therefore the status should be changend from "broken" or "ingame" to "Playable", but with note in the Wikia. All games, which were running on the servers of Gamespy are offline too, but this is the problem of dev, not PCSX2.

(PS: The game servers have shut down, but the game itself is playable.)

Games, which are involved:
http://wiki.pcsx2.net/index.php/EverQuest_Online_Adventures:_Frontiers
http://wiki.pcsx2.net/index.php/EverQuest_Online_Adventures

**Comment by ssakash** [comment]
does the game revolve around using the specific severs (or) could it be also played offline ?
this seems like a little different case relating to the definition of the Playable state.

**Comment by trader5896** [comment]
As I've seen is that the Game works, but the Servers don't work. It's "Playable" by the emu itself, but because the servers are shutdown, it doesn't work. PCSX2 is perfectly emulating the PS2, which cannot play that game either, because the servers shutdown.

**Comment by trader5896** [comment]
I didn't test the game, but this is not possible anyway.

**Comment by trader5896** [comment]
Playable: You can get from 'new game' to 'end credits' (regardless of speed).

This definition is not very correct, because playable should be: You can emulate the PS2 with or not glitches to the end.

**Comment by ssakash** [comment]
even if you feel it's incorrect, the definition can't be changed unless you talk about this to Bositman.

**Comment by trader5896** [comment]
This is right, Bositman will decide. I think he will see this thread I think..

**Comment by trader5896** [comment]
The definition of playable is very good, but for games, which have no end, this will not work and 3rd party problems like servers for example Gamespy is not actually the problem of the Pcsx2 or Ps2... 

And Games like Mortal Kombat have no end to be honest, except the story mode campaign. And what about games, which go from start to end, but the extra part (mostly Minigames) doesn't work. 


Therefore a precise definition would be great, but in general the definition is good.

**Comment by ssakash** [comment]
that's a valid point. but, it would be better if you could discuss about this in the forum.

**Comment by trader5896** [comment]
I'll wait for first. Bositman will read it anyway, there's no hurry. 

I will first export all, which I regard as a bug and then Bositman can fix or close.

**Comment by bositman** [comment]
The definition should be simple and clear. We can't include every single game in a definition without making it huge and more specific. From start of game to end credits means from start to finish, if the game is a fighter it means it just works with all characters. If there are mini games that are not required to finish the game, even if they don't work the game is still considered playable.
Games that require online connectivity can't be tested actually, so we can't know their status.

**Comment by jensdraht1999** [comment]
@bositman 

Hi, I wanted to report on this again. You can actually run both games:
http://wiki.pcsx2.net/index.php/EverQuest_Online_Adventures:_Frontiers
http://wiki.pcsx2.net/index.php/EverQuest_Online_Adventures

Check here:
https://www.projectreturnhome.com/

Both games run with PCSX2. Look at the videos. 
Even, before the servers were shutdown in 2012, they were running.

You can write in the wiki or the compatibility list, that they are running, but that the original servers were shutdown in 2012, so you will not be able to play the game back in the days, but this is not an emulation side problem. 

Basically two games less, that do not work with PCSX2, only 20 games left that are either Menu or Ingame.

**Comment by bositman** [comment]
@jensdraht1999 You are welcome to create a test report in our forum which is responsible for updating the compatibility status of the list: 
[](https://forums.pcsx2.net/Thread-Everquest-Online-Adventures-SCES-51725-E--29542)
[](https://forums.pcsx2.net/Thread-EverQuest-Online-Adventures-Frontiers-SLUS-20744-U)
